app/views/users/show.html.erb

```diff
@@ -0,0 +1,29 @@
+<div class='post-container'>
+    <div class='post-img'>
+        <img src='https://i.pinimg.com/originals/73/16/f5/7316f550de9ca0045e3d8d98a5bb5e44.png' alt='post' >
+    </div>
+    <div class='post-wrapper'>
+        <h1> <%= @user.name %> </h1>
+        <p> Number of posts: <%= @user.posts_counter %> </p>
+    </div>
+</div>
+
+<div class='bio-div'>
+    <h4> Bio</h4>
+    <p> <%= @user.bio %> </p>
+
+</div>
+
+<% @posts.each do |post| %>
+    <%= link_to user_posts_path(@user.id) do %>
+        <div class='post-div'>
+            <h4> <%= post.title %> </h4>
+            <p> lorem ipsum deprocadi ips deasdflorem ipsum deprocadi ips deasdflorem ipsum deprocadi ips deasdflorem ipsum deprocadi ips deasdflorem ipsumdeasdflorem ipsum deprocadi ips deasdflorem ipsum ... </p>
```

Comment:
- [ ] Let's not hard code this. It should be the actual text from the post. Just like you've displayed `post.text`, please also display `post.text` in this `<p>` tag.